golang/cosmos/x/vtransfer/keeper/keeper.go

```diff
@@ -180,6 +280,14 @@ func (k Keeper) InterceptOnTimeoutPacket(
 	if err != nil {
 		return err
 	}
+
+	origPacket := packet
+	packetStore, packetKey := k.PacketStoreFromOrigin(ctx, types.PacketSrc, packet)
+	if packetStore.Has(packetKey) {
+		origPacket.Data = packetStore.Get(packetKey)
```

Comment:
I'm confused, why do we need to restore the original packet's data? Wouldn't a received timeout packet contain the original packet data in the first place?